nsqadmin/http.go

```diff
@@ -1036,24 +1036,65 @@ type counterStats struct {
 }
 
 type rankStats struct {
-	TopicName         string `json:"topic_name"`
-	TotalChannelDepth int64  `json:"total_channel_depth"`
-	MessageCount      int64  `json:"message_count"`
-	HourlyPubSize     int64  `json:"hourly_pubsize"`
+	Name         string 	 `json:"name"`
```

Comment:
as rank of channel timeout&requeue added. I woule like to give a more general name.